tools/cloud/demo.sh

```diff
@@ -448,6 +493,7 @@ if ((FLEET_SIZE==1)); then
         --decoder-manifest-arn ${DECODER_MANIFEST_ARN} \
         --association-behavior ValidateIotThingExists \
         --model-manifest-arn ${MODEL_MANIFEST_ARN} \
+        --attributes '{"Vehicle.OBD.Color":"Red"}' \
         --vehicle-name "${VEHICLE_NAME}" | jq -r .arn
 else
```

Comment:
Add `--attributes` to the multi-vehicle creation below

Reply:
thanks good catch